### PR TITLE
[W5.10h][F09-A4] Md Azhar

### PR DESCRIPTION
**Pull request by md-azsa**
Learning Outcomes - Use Cases
Added a new use case to the developerGuide markdown for case,
Renaming an existing tag

### CODE REVIEW

**Comment by eldon-chung** [comment]
very comprehensive use cases

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -87,6 +87,36 @@ Use case ends.
...
+Use case ends.
+
+**Extensions**
+
+2a. Tag to rename does not exist
+
+> 2a1. AddressBook shows an error message <br>
+  Use case resumes at step 2
+
+3a. The list is empty
```
Comment (k-l-a) [inline doc/DeveloperGuide.md:112]: This doesn't seem like an extension of step 3. Perhaps it is earlier than that?

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -87,6 +87,36 @@ Use case ends.
...
+
+> Use case ends
+
+4a. The given index is invalid
```
Comment (k-l-a) [inline doc/DeveloperGuide.md:116]: Similar here. The extension should be numbered on the step in which the exceptional event can occur.

**Review by k-l-a — COMMENTED** [review]
@md-azsa Some comments added. Please close PR after reading.

**Pull request closed without merging**